<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>18</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Brush"/>
    <addaction name="action_Pen"/>
    <addaction name="action_Fill_The_Trusty_Bucket"/>
    <addaction name="action_Gradient"/>
    <addaction name="action_Image_Fill"/>
   </widget>
   <widget class="QMenu" name="menu_Draw">
    <property name="title">
     <string>&amp;Draw</string>
    </property>
    <addaction name="actionLine"/>
    <addaction name="actionRect"/>
    <addaction name="actionElipse"/>
    <addaction name="actionPolygon"/>
   </widget>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Draw"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Pen"/>
   <addaction name="action_Brush"/>
   <addaction name="action_Fill_The_Trusty_Bucket"/>
   <addaction name="action_Gradient"/>
   <addaction name="action_Image_Fill"/>
   <addaction name="separator"/>
   <addaction name="actionLine"/>
   <addaction name="actionRect"/>
   <addaction name="actionElipse"/>
   <addaction name="actionPolygon"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_Open">
   <property name="icon">
    <iconset>
     <normaloff>:/open.png</normaloff>:/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset>
     <normaloff>:/new.png</normaloff>:/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset>
     <normaloff>:/save.png</normaloff>:/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset>
     <normaloff>:/save_as.png</normaloff>:/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset>
     <normaloff>:/exit.png</normaloff>:/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Brush">
   <property name="icon">
    <iconset>
     <normaloff>:/brush1.png</normaloff>:/brush1.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Brush</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_Pen">
   <property name="icon">
    <iconset>
     <normaloff>:/pencil.png</normaloff>:/pencil.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Pen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Fill_The_Trusty_Bucket">
   <property name="icon">
    <iconset>
     <normaloff>:/fill--background-color-png-image-58965.png</normaloff>:/fill--background-color-png-image-58965.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Fill &quot;The Trusty Bucket&quot;</string>
   </property>
  </action>
  <action name="action_Gradient">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/img_529565.png</normaloff>:/img_529565.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Gradient</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_Image_Fill">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icon-image-512.png</normaloff>:/icon-image-512.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Image Fill</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Editing-Line-icon.png</normaloff>:/Editing-Line-icon.png</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="actionRect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Editing-Rectangle-icon.png</normaloff>:/Editing-Rectangle-icon.png</iconset>
   </property>
   <property name="text">
    <string>Rect</string>
   </property>
  </action>
  <action name="actionElipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/shape_ellipse-512.png</normaloff>:/shape_ellipse-512.png</iconset>
   </property>
   <property name="text">
    <string>Elipse</string>
   </property>
  </action>
  <action name="actionPolygon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/software_shape_polygon-512.png</normaloff>:/software_shape_polygon-512.png</iconset>
   </property>
   <property name="text">
    <string>Polygon</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res_paint/loadout.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
